Makefile

```diff
@@ -173,10 +173,10 @@ add-tag:
 push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
-	@git push upstream ${TAG}
+	@git push git@github.com:open-telemetry/opentelemetry-collector.git ${TAG}
```

Comment:
Huh... I didn't know this worked. Thanks for sharing!

Reply:
haha